libnavigation-copilot/proguard-rules.pro

```diff
@@ -0,0 +1 @@
+-keepnames class com.mapbox.api.directions.v5.models.** {*;}
```

Comment:
It seems like it would be better to put this line into the consumer file of the original SDK. 
Questions:
1) Is it possible for Java, not Android?
2) Why do you keep only the models package?
3) Why do you need to keep that at all?